config/tasklists/get-a-divorce.json

```diff
@@ -0,0 +1,336 @@
+{
+  "title": "Get a divorce: step by step",
+  "base_path": "/get-a-divorce",
+  "description": "<p>How to file for divorce if you’re in England or Wales.</p><p>There is a different process if you’re <a href=\"/divorce/respond-to-a-divorce-petition\">responding to a divorce petition</a>, or if you’re in <a href=\"https://www.scotcourts.gov.uk/taking-action/divorce-and-dissolution-of-civil-partnership\">Scotland</a> or <a href=\"https://www.nidirect.gov.uk/articles/getting-divorcedissolution-civil-partnership\">Northern Ireland</a>.</p>",
+  "links": {
+    "breadcrumbs": [
+      {
+        "title": "Home",
+        "url": "/"
+      },
+      {
+        "title": "Births, deaths, marriages and care",
+        "url": "/browse/births-deaths-marriages"
+      },
+      {
+        "title": "Marriage, civil partnership and divorce",
+        "url": "/browse/births-deaths-marriages/marriage-divorce"
+      }
+    ]
+  },
+  "related_paths": [
```

Comment:
@sihugh I've added this so we no longer need to rely on several constants with base paths, the json file should dictate this.

Reply:
The primary paths are extracted from the JSON structure too, here: https://github.com/alphagov/govuk_navigation_helpers/pull/90/files#diff-66b229b71113b93e5d2264bddcf2caa2R23

Reply:
Good plan Stan